com.unity.uiwidgets/Runtime/engine2/AndroidPlatformUtil.cs

```diff
@@ -1,11 +1,11 @@
-﻿#if !UNITY_EDITOR && UNITY_ANDROID
+﻿#if UNITY_ANDROID
```

Comment:
isn't AndroidPlatformUtil Android player only ? if it is, why removing !UNITY_EDITOR?